components/avatar/avatar.css

```diff
@@ -0,0 +1,51 @@
+@import '../../homepage/homepage.css';
```

Comment:
Avoid repetition
I would suggest importing `homepage.css` only once in the file you find it to be most important.
In this case, it would be better fit to be in avatar.css file so that it can be used in badge.css also. Then, you won't have to import it again in badge.css as avatar is being used in badge.